Andela-Ray/PropertyPro-lite
# #167243340 Add JWT authentication

**What does this PR do?**
Adds JWT authentication to our API

**Description of Task to be completed?**
-Add helper to create JWT tokens for newly signed-up and logged in users
-Add middleware to authenticate JWT tokens on protected routes

**How should this be manually tested?**
Use Postman to test out protected endpoints hosted at 'https://andelaray-propertypro-lite.herokuapp.com/' such as:
-post property/
-patch property/:id
-patch property/:id/sold
-delete property/:id
Your requests should only succeed if you pass the token you received on signing-up

**What are the relevant pivotal tracker stories?**
#167243340
